keywording/text.xml

```diff
@@ -434,6 +434,36 @@ keyword, so packages are not to be marked stable for one of these architectures.
 </body>
 </subsection>
 
+<subsection>
+<title>Keeping track of pending stabilizations</title>
+<body>
+
+<p>
+Maintainers need some method or system to organize and start pending
+stabilizations once they become due.
+</p>
+
+<p>
+There are several options available:
```

Comment:
One could read this as being the only options, which isn't the case. For example, I keep track of my version bumps in an old-fashioned list with Emacs' org-mode. :smiley:

Maybe something like the following instead?
```suggestion
There are several tools available that can help with this:
```